Copy edit and suggestions for the writing handbook (#1082)
GoogleChrome/web.dev: src/site/content/en/handbook/effective-instruction/index.md

  - mfriesenhahn
date: 2019-06-26
description: |
  Learn strategies for writing effective instruction on web.dev, especially for adult learners.
---

Most web.dev content includes at least some instruction. These guidelines help make your instruction easy to follow.

Explain _why_ to do something before explaining _how_ to do it. Knowing the end goal helps readers understand the purpose of individual steps.

> Keep the scope of instruction manageable. If a task is complex, find ways to chunk the steps. For example, provide the essential pieces in a post and put the ancillary pieces in one or more codelabs.

Author: The for example bit doesn't tell me more. I'm still not sure what info is essential vs ancillary. Can we have a practical example of good/bad?

Also, ancillary seems like an overly complex word.

Reviewer: 'end goal' -> 'goal' (Shrunk and White II.17: Omit needless words.)

Reviewer 2: I'm not sure that we can or should provide criteria for determining whether content is supplemental to the main idea of a piece of content. Seems like those criteria would be highly dependent on context? Did you have some in mind? Or are you leaning toward cutting this as unhelpfully general?

Always provide examples for complex or abstract concepts. If you're debating whether to give an example, add one.

> Use web.dev's built-in [Instruction components](/handbook/web-dev-components#instruction) for common tasks that are often referenced on the site (for example, instructions about opening Glitch or Chrome DevTools). This approach provides a consistent user experience and makes it easy to update instructions across the site when needed.

Author: I like that we explain the why here (it follows the handbook principles to explain things). Should we do this more often when we list style rules? Or is it clear enough why something is a rule?

Reviewer 2: This is something I was struggling with--balancing robust explanation and skimmability to support different purposes for reading.

If we had a progressive disclosure control whose salience worked a bit better for this context than the `detail` widget we have now, I'd definitely provide more explanations. (Basically, I want something like the [Material Design Writing](https://material.io/design/communication/writing.html#principles) page.)

I could perhaps work on that with @robdodson if you're willing to flag cases where you think additional explanation or rationales would be helpful.

Reviewer 2: Also, mostly really like the edit, which solves some phrasing problems I was having trouble with, but would like to keep a reference to tools if possible since that really is the limiting factor--we have instructions for Glitch and DevTools because those are the tools referenced most often in posts.

## Instruction for adult learners
Adult learners bring a lot experience. It's important to honor that experience while acknowledging the challenges of the tasks you're teaching. 

> Be mindful of [voice](/handbook/voice). To avoid alienating your readers, try to strike a balance between showing expertise and humility.

Author: Is the balance between expertise/humility covered in voice? This is such a short sentence, that I'm almost not sure how to do it. Examples would be helpful so I don't have to jump off to a link to find out what this means

Reviewer 2: Yeah, agree about needing examples and non-examples. I'm thinking it might be good to ask tech writers to be on the lookout while they're reviewing content so the examples can be authentic.

> Avoid words and phrases that make it sound like a concept or task is easy or obvious—it may not be! For example, the following words and phrases can frustrate readers when a task turns out to be harder: _obviously_, _simply_, _of course_, _clearly_, _just_, _everyone knows_, and _easy_.

Reviewer 2: ```suggestion
Avoid words and phrases that make it sound like a concept or task is easy or obvious—it may not be! For example, the following words and phrases can frustrate readers when they find a task difficult: _obviously_, _simply_, _of course_, _clearly_, _just_, _everyone knows_, and _easy_.
```
To emphasize that difficulty is relative and to avoid using a comparative adjective without something to compare it to.
